### PR TITLE
Add the pull request template

### LINKED ISSUE #4622: Look into better ways to ensure content parity
[the content parity rule for translations](https://osu.ppy.sh/wiki/en/Article_Styling_Criteria#content-parity) is written with the right idea in mind, but in practice it gets messy when the original English article isn't fully up-to-date or written well. translators (rightly) choose to correct the mistakes and deviate from English a bit, leading to [situations the rule was designed to prevent](https://discord.com/channels/188630481301012481/218677502141399041/788156447375622216) if they don't also update the English article.

imo it's a problem that:

- we are effectively making translators responsible for maintaining English articles, which at best is not what they want to spend their time doing, and at worst is impossible because they may not have the required English writing skills
- deviations from English can easily go unnoticed, since many languages have little to no reviewers that are experienced with osu-wiki
- following English as close as possible (according to the rule) means translations adopt plenty of bad writing, structure, and formatting from English, basically just putting off work for far-future contributors to deal with

I don't know if there's any good way to prevent these problems, just opening this issue for now to see if anyone else has had thoughts about this

### PR DESCRIPTION
**Pull request by TicClick**
now that we have a quick checklist to go through, this closes #4622 (first attempt: #4749)

### CODE REVIEW

**Comment by TicClick** [comment]
I'm considering linking to the [files on GitHub](https://github.com/ppy/osu-wiki/tree/master/wiki/en/osu!_wiki/Contribution_guide/en.md#self-check) instead:
- surprisingly, they are easier to read and perceive, at least for me
- no ci failures

**Comment by TicClick** [comment]
this should sound better (and look, too)

**Review by peppy — DISMISSED** [review]
Willing to give this a try. Not hopeful that it's going to improve anything (as I said, we tried this on the other development repositories and eventually removed it as it was both not used and annoying).

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  - Use [x] to complete the items
+  - Remove the items unrelated to your work
+  - Add any relevant information you consider useful
+  - If there are no reviewers for your language, please mention it explicitly
+-->
+
+# Self-check
+
+- [ ] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
+- [ ] *(translations only)* The changes are reviewed [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
```
[the remark this answers is not in the document]
Comment (TicClick) [reply .github/pull_request_template.md:11]: it did, yeah. can probably be fixed by making it "The changes are reviewed **on GitHub**"

**Review by cl8n — APPROVED** [review]
same thoughts as https://github.com/ppy/osu-wiki/pull/6237#pullrequestreview-782785873

**Pull request merged**